### PR TITLE
Fix ukrainian characters

### PR DESCRIPTION
**Pull request by Djaler**

### CODE REVIEW

File: uk.json [hunk 1]
```diff
@@ -13,9 +13,10 @@
   "{": "Х",
   "]": "ї",
   "}": "Ї",
-  "|": "/",
-  "`": "ё",
-  "~": "Ё",
+  "\": "ґ",
```
Comment (ai) [inline uk.json:16]:
```suggestion
  "\\": "ґ",
```

Comment (Djaler) [reply uk.json:16]: Oh, my mistake

**Comment by Djaler** [comment]
Done

**Comment by ai** [comment]
Thanks. Released in 0.8.2.

**Comment by Djaler** [comment]
May I ask you to add hacktoberfest topic to the repo?

**Comment by ai** [comment]
Done

**Pull request merged**